_docs/sysadmin/configuration/student_photos.md

```diff
@@ -8,11 +8,14 @@ redirect_from:
 
 In a large class, it can be difficult for instructors, full-access graders, and
 limited-access graders to learn the names of their students.  The Student Photos page is accessed from
-a blue button at the top of the course main page.  Instructors can upload photos of
-their students and view all uploaded students’ photos. Full-access graders and
-limited-access graders also have access to this page. Full-access graders can
-see photos of all students in the course, while limited-access graders can only see the photos of
-the students in their assigned registration sections.
+the "Student Photos" tab on the sidebar located on the course's pages.  
+
+![](/images/student_photos_sidebar_tab.png)
```

Comment:
I felt like the description seemed sufficient and since the image is cropped it doesnt really tell where the button is located.

Reply:
Do you think it would be better to use an image that displays the entire page and points out where the button is, or was the description good enough that providing an image is unnecessary?

Reply:
Only  having the description seems good enough for me

Reply:
Okay.  I'll remove the image.